examples/purchaseTesterTypescript/app/screens/FooterPaywallScreen.tsx

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
-import { PaywallFooterView } from 'react-native-purchases-ui';
+import {PaywallFooterView} from 'react-native-purchases-ui';
 
-import { Text, View } from 'react-native';
-import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import RootStackParamList from "../RootStackParamList";
+import {StyleSheet, View} from 'react-native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import RootStackParamList from '../RootStackParamList';
 
 type Props = NativeStackScreenProps<RootStackParamList, 'FooterPaywall'>;
 
 const FooterPaywallScreen: React.FC<Props> = () => {
+  const styles = StyleSheet.create({
+    flex1: {
+      flex: 1,
+    },
+  });
+
   return (
-    <View style={{
-      justifyContent: 'space-between',
-      flexDirection: 'column',
-      flexGrow: 1,
-      backgroundColor: '#4b72f6',
-    }}>
-      <View style={{ backgroundColor: '#ff0000'}}>
-        <Text>Top</Text>
-      </View>
-      <PaywallFooterView/>
+    <View style={styles.flex1}>
```

Comment:
I ended up having to add flex 1 here too, since otherwise the view would render perfectly fine, but it wouldn't respond to taps, which was super weird. It's like it was the correct size but react-native still treated it as size zero. 

flex 1 works like a charm, though.

Reply:
If I remove this the view doesn't render at all. If I understand correctly the React Native docs, the reason why it doesn't render is because it needs a size:

> A component can only expand to fill available space if its parent has dimensions greater than 0. If a parent does not have either a fixed width and height or flex, the parent will have dimensions of 0 and the flex children will not be visible.

https://reactnative.dev/docs/height-and-width#flex-dimensions

Setting it to `flex: 1` or `width: '100%', height: '100%'` fixes it, but I think that's something devs have to do anyways?

Reply:
you're right. I feel like we're going to have to go with a container so we can control the usage